pandas/formats/format.py

```diff
@@ -2138,6 +2016,31 @@ def formatter(value):
 
         return formatter
 
+    def _get_formatted_data(self):
+        """
+        Returns formatted float values. This is different from
```

Comment:
Added note to `format.py` - unfortunately, integrating the no-format option from my new function into `get_result_as_array` caused a whole slew of test failures.  So I just moved it into `to_native_types` instead.